frontend/src/app/testResults/submissions/Submission.tsx

```diff
@@ -51,6 +52,9 @@ const Submission = () => {
             >
               Upload history
             </LinkWithQuery>
+            <div>
+              <h1 className="margin-top-2">Submission detail</h1>
+            </div>
```

Comment:
```suggestion
            <div>
              <h1 className="margin-top-2">Submission details</h1>
            </div>
```
Added an "s," otherwise looks good to me. We may want to change `Report ID` to `Submission ID` in the "Upload history" page for consistency, but that's a fight for another day.

Reply:
Will make the ID changes tomorrow